### PR TITLE
Fix upload date field

### LINKED ISSUE #1457: Bad upload date localization
The upload date field is showing a wrong language (maybe similar to #1215):
How? setting **Default content country** to any non-english language (in my case spanish).

Screenshot of **Trending** showing the _upload date_ in english:

Search results also too, but in arab:

### PR DESCRIPTION
**Pull request by kapodamy**
Implement correctly **Accept-Language** HTTP header, also require some changes on **NewPipeExtractor** to work (https://github.com/TeamNewPipe/NewPipeExtractor/pull/91).

Fixes #1457 #1215

✔ I carefully read the contribution guidelines and agree to them.

### CODE REVIEW

**Comment by theScrabi** [comment]
I quickly looked over the PR on the extractor.

I think the downloader should be stateless. Putting the language code there does not seem to be a good idea. I think it was better to set it in the newpipe service and then just add a language/locale code to the downloader.download() methode.

File: app/src/main/java/org/schabi/newpipe/Downloader.java [hunk 1]
```diff
@@ -40,10 +40,13 @@
 
 public class Downloader implements org.schabi.newpipe.extractor.Downloader {
     public static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0";
+    private static final String ACCEPT_LANGUAGE = "Accept-Language";
 
     private static Downloader instance;
     private String mCookies;
     private OkHttpClient client;
+    private static String countryCode = "GB";
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/Downloader.java]: Still not stateless. I think the country, and language should be controlled by the back end, not by the downloader, which is part of the frontend.

**Comment by theScrabi** [comment]
Please squash fix1 and fix2 and give it a more meaningfull name :)

**Comment by lf-** [comment]
Some of the comments in this contribution could be improved. For instance, the reference to line numbers will be unhelpful as soon as that file is modified, so it should be referring to something more general or just be self contained in the first place. 

Also, there is a commented out line of code which should probably get deleted.

**Pull request closed without merging**